Mathlib/Algebra/GroupWithZero/Action/Hom.lean

```diff
@@ -0,0 +1,58 @@
+/-
+Copyright (c) 2025 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+
+import Mathlib.Algebra.GroupWithZero.Action.Defs
+import Mathlib.Algebra.Group.Hom.Instances
+
+/-! # Zero-related `•` instances on group-like morphisms -/
```

Comment:
Do you want to add some `## Tags` section for searchability?

Reply:
`Mathlib/Algebra/Group/Hom/Instances.lean` doesn't have any so I'm not that interested in doing so.